### ISSUE TITLE
AVT 1 - React Slider has DAP violation

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3061]
## Environment
macOS Mojave 10.14.5 
Chrome Version 75.0.3770.80 (Official Build) (64-bit)
Dynamic Assessment Plugin (DAP)

## Detailed description
Run DAP and there is one violation (see screenshot):
<img width="1119" alt="Screen Shot 2019-06-13 at 10 50 17 AM" src="https://user-images.githubusercontent.com/21676914/59448381-9c7f2800-8dca-11e9-8b90-cd53052a3114.png">

To fix this issue add a label to aria-label, such as aria-label="Label for slider value"
